applet.js

```diff
@@ -804,9 +792,9 @@ MyApplet.prototype = {
   }
 
 , weatherUrl: function weatherUrl() {
-    let output = QUERY_URL + ' where woeid="' + this._woeid + '" and u="' + this.unitToUrl() + '"'
+    //let output = QUERY_URL + ' where location="' + this._woeid + '" and u="' + this.unitToUrl() + '"'
     //let output = QUERY_URL + this._woeid + '_' + this.unitToUrl() + '.xml"'
-    return output
+    return 'https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20weather.forecast%20where%20woeid%20%3D%20%22' + this._woeid + '%22&format=json&env=store%3A%2F%2Fdatatables.org%2Falltableswithkeys'
```

Comment:
@yoosefi Did you find some documentation for this?  Where did the `env=store://datatables.org/alltableswithkeys` bit come from?